packages/client/src/rx.ts

```diff
@@ -9,9 +12,7 @@ export class ObservableAbortError extends Error {
 }
 
 /** @internal */
-export function observableToPromise<TValue>(
-  observable: Observable<TValue, unknown>,
-) {
+export function observableToPromise<TValue>(observable: Observable<TValue>) {
```

Comment:
Note: This whole function can be replaced with RxJS's `firstValueFrom`.

Reply:
Even the abort functionality?